05 - coder-files/src/App.jsx

```diff
@@ -1,16 +1,37 @@
+import { useEffect, useState } from "react";
+import Card from "./components/Card";
+
 export default function App() {
+  const [users, setUsers] = useState([]);
+  const [change, setchange] = useState(false);
+
+  useEffect(() => {
+    fetch("https://random-data-api.com/api/users/random_user?size=12")
+      .then((res) => res.json())
+      .then((data) => {
+        console.log(data[0]);
+        setUsers(data);
+      })
+      .catch((err) => console.log(err));
+  }, [change]);
+
+  const displayUsers = users.map((user) => <Card key={user.id} {...user} />);
+
   return (
-    <div className="flex gap-10 justify-center flex-wrap px-10 text-blue-600 mt-10">
-      <a href="/">Home</a>
-      <a href="/" target="_blank">Exercise 1</a>
-      <a href="/" target="_blank">Exercise 2</a>
-      <a href="/" target="_blank">Exercise 3</a>
-      <a href="/" target="_blank">Exercise 4</a>
-      <a href="/" target="_blank">Exercise 5</a>
-      <a href="/" target="_blank">Exercise 6</a>
-      <a href="/" target="_blank">Exercise 7</a>
-      <a href="/" target="_blank">Exercise 8</a>
-      <a href="/" target="_blank">Exercise 9</a>
+    <div className="flex justify-center min-h-screen bg-[#E1F6DC]">
+      <div className="grid gap-5 sm:gap-10 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 my-10 px-5">
+        <div className="sm:col-span-2 lg:col-span-3 xl:col-span-4 text-center">
+          <button
+            className="bg-[#A30036] rounded-md sm:rounded-xl w-full sm:w-auto"
+            onClick={() => setchange((prevChange) => !prevChange)}
```

Comment:
why do you need previous change, you do not need a function in setchange, you can  just do like`setchange(!change)`